front/pages/api/w/[wId]/assistant/agent_configurations/[aId]/index.ts

```diff
@@ -97,35 +97,33 @@ async function handler(
         });
       }
 
-      const maxToolsUsePerRun =
-        bodyValidation.right.assistant.maxToolsUsePerRun;
+      const maxStepsPerRun = bodyValidation.right.assistant.maxStepsPerRun;
```

Comment:
We need to send both previous and new fields on the AssistantBuilder form to not error for all people that are in the AssistantBuilder when you deploy this new code :/

Reply:
Indeed :/